src/groups/mqb/mqbi/mqbi_queue.h

```diff
@@ -964,12 +965,13 @@ class QueueHandleFactory {
     /// Create a new handle, using the specified `allocator`, for the
     /// specified `queue` as requested by the specified `clientContext` with
     /// the specified `parameters`, and associated wit the specified
-    /// `stats`.
+    /// `stats`.  Use the specified 'counterOfUnconfirmed'.
```

Comment:
No argument `counterOfUnconfirmed` here now, only the `parent` which I would like to rename to avoid confusion